tapeagents/view.py

```diff
@@ -151,9 +150,9 @@ def put_new_view_on_stack(self, step):
         self.messages_by_agent[receiver].append(step)
 
     @staticmethod
-    def compute(tape: Tape) -> TapeViewStack[StepType]:
+    def compute(tape: Tape, agent_name: str = "root") -> TapeViewStack[StepType]:
```

Comment:
I'd like to be slightly more explicit here

```suggestion
    def compute(tape: Tape, root_agent_name: str = "root") -> TapeViewStack[StepType]:
```